packages/web/src/components/incentivize/pool-incentivize-details/PoolIncentivizeDetails.tsx

```diff
@@ -10,8 +10,8 @@ import { getDateUtcToLocal } from "@common/utils/date-util";
 import dayjs from "dayjs";
 import { useGnotToGnot } from "@hooks/token/use-gnot-wugnot";
 import MissingLogo from "@components/common/missing-logo/MissingLogo";
-import Tooltip from "@components/common/tooltip/Tooltip";
-import IncentivizeBlockTooltipContent from "../incentivize-block-tooltip-content/IncentivizeBlockTooptipContent";
+// import Tooltip from "@components/common/tooltip/Tooltip";
+// import IncentivizeBlockTooltipContent from "../incentivize-block-tooltip-content/IncentivizeBlockTooptipContent";
```

Comment:
Please remove unused lines.

Reply:
I think should be leave it there if we need to comback with block calculation